cloudify_hostpool/storage/sqlite.py

```diff
@@ -0,0 +1,123 @@
+# #######
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+import json
+import sqlite3
+from base import AbstractStorage
+
+
+class SQLiteStorage(AbstractStorage):
+    """ Storage wrapper for SQLite DB implementing AbstractStorage interface"""
+    _CREATE_TABLE = 'CREATE TABLE IF NOT EXISTS servers ' \
+                    '(global_id integer PRIMARY KEY, server_id text, ' \
+                    'address text, auth text, port integer, alive integer, ' \
+                    'reserved integer)'
+
+    def __init__(self, db_filename):
+        self._filename = db_filename
+        self._create_table()
+
```

Comment:
place public methods at the top and private methods at the bottom...its just the way we try to do it. public methods are more interesting to look and so its nicer if they are the first thing to see :)